app/config/services/platform.xml

```diff
@@ -368,6 +369,12 @@
             <argument type="service" id="app.repository.citizen_action"/>
         </service>
 
+        <service autowire="true" id="AppBundle\CitizenAction\CitizenActionCommandHandler">
+            <argument key="$mailer" type="service" id="app.mailer.transactional" />
```

Comment:
Why not creating an alias for `app.mailer.transactional` and then use autowiring ?